packages/Filter/src/components/Item/Item.js

```diff
@@ -123,9 +122,16 @@ function Item(props) {
     switch (selectedColumnType) {
       case types.columnTypes.BOOLEAN:
         return (
-          <sc.ValueInput as={Select} value={value.toString()} onChange={handleChangeBooleanFilterValue}>
-            <option value="true">{I18n.t("filter.rules.true")}</option>
-            <option value="false">{I18n.t("filter.rules.false")}</option>
+          <sc.ValueInput data-pka-anchor="filter.item.valueInput" value={value.toString()}>
```

Comment:
Each time I review this switch, I want to comment about turning it into an object, but then I realize that each of these statements would end having a different signature if they were turned into a function. I think this is one of the few cases where I find that the switch statement performs better than using an object pattern.

:)